exercises/list-ops/src/example/java/ListOps.java

```diff
@@ -0,0 +1,56 @@
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.function.BiFunction;
+import java.util.function.BinaryOperator;
+import java.util.function.Predicate;
+import java.util.function.UnaryOperator;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class ListOps {
+
+    private ListOps() {
+    }
+
+    public static <T> int length(final List<T> list) {
+        return list.size();
+    }
+
+    public static <T> List<T> reverse(final List<T> list) {
+        List<T> result = new ArrayList(list);
+        Collections.reverse(result);
+        return result;
+    }
+
+    public static <T> List<T> map(final List<T> list,
+            UnaryOperator<T> mapper) {
+        return list.stream().map(mapper).collect(Collectors.toList());
```

Comment:
Tried to keep it simple. The example is needed only to test the test. No pun intended. :)